include/npcomp/Dialect/ATen/ATenOps.td

```diff
@@ -0,0 +1,766 @@
+      AnyTensor:$other,
+      AnyScalar:$alpha
+  );
+  let summary = "aten add_ operator";
+  let description = [{
+    AddUnderOp
+    aten add_ operator
+  }];
+  let extraClassDeclaration = [{
+    std::map<std::string, uint64_t> getStatistics();
+  }];
+}
+
+def aten_AsStridedOp: aten_Op<"as_strided", [NoSideEffect, StatisticsOpInterface]>,
+    Results<(outs AnyTensor)> {
+  let arguments = (
+  ins AnyTensor:$self,
+      AnyType:$size,
+      AnyType:$stride
+  );
+  let summary = "aten as_strided operator";
+  let description = [{
+    AsStridedOp
+    aten as_strided operator
+  }];
+  let extraClassDeclaration = [{
+    std::map<std::string, uint64_t> getStatistics();
+  }];
+}
+
+def aten_ConvolutionOverrideableOp: aten_Op<"convolution_overrideable", [NoSideEffect, StatisticsOpInterface]>,
+    Results<(outs AnyTensor)> {
+  let arguments = (
+  ins AnyTensor:$input,
+      AnyTensor:$weight,
+      AnyTensor:$bias,
+      AnyType:$stride,
+      AnyType:$padding,
+      AnyType:$dilation,
+      AnyScalar:$transposed,
+      AnyType:$output_padding,
+      AnyScalar:$groups
+  );
+  let summary = "aten convolution_overrideable operator";
+  let description = [{
+    ConvolutionOverrideableOp
+    aten convolution_overrideable operator
+  }];
+  let extraClassDeclaration = [{
+    std::map<std::string, uint64_t> getStatistics();
+  }];
+}
+
+def aten_ConvolutionBackwardOverrideableOp: aten_Op<"convolution_backward_overrideable", [NoSideEffect, StatisticsOpInterface]>,
+    Results<(outs AnyTensor, AnyTensor, AnyTensor)> {
+  let arguments = (
+  ins AnyTensor:$grad_output,
+      AnyTensor:$input,
+      AnyTensor:$weight,
+      AnyType:$stride,
+      AnyType:$padding,
+      AnyType:$dilation,
+      AnyScalar:$transposed,
+      AnyType:$output_padding,
+      AnyScalar:$groups
+  );
+  let summary = "aten convolution_backward_overrideable operator";
+  let description = [{
+    ConvolutionBackwardOverrideableOp
+    aten convolution_backward_overrideable operator
+  }];
+  let extraClassDeclaration = [{
+    std::map<std::string, uint64_t> getStatistics();
+  }];
+}
+
+def aten_DivOp: aten_Op<"div", [NoSideEffect, StatisticsOpInterface]>,
+    Results<(outs AnyTensor)> {
+  let arguments = (
+  ins AnyTensor:$self,
+      AnyTensor:$other
+  );
+  let summary = "aten div operator";
+  let description = [{
+    DivOp
+    aten div operator
+  }];
+  let extraClassDeclaration = [{
+    std::map<std::string, uint64_t> getStatistics();
+  }];
+}
+
+def aten_DivUnderOp: aten_Op<"div_", [NoSideEffect, StatisticsOpInterface]>,
+    Results<(outs AnyTensor)> {
+  let arguments = (
+  ins AnyTensor:$self,
+      AnyTensor:$other
+  );
+  let summary = "aten div_ operator";
+  let description = [{
+    DivUnderOp
+    aten div_ operator
+  }];
+  let extraClassDeclaration = [{
+    std::map<std::string, uint64_t> getStatistics();
+  }];
+}
+
+// def aten_DivOp: aten_Op<"div", [NoSideEffect, StatisticsOpInterface]>,
```

Comment:
Why commented out?

Reply:
It's an artifact of the generation process.  IIRC there's an overload in what pytorch exports?